src/main/java/se/thinkware/gocd/dockerpoller/PackageRepositoryPoller.java

```diff
@@ -210,22 +211,30 @@ public PackageRevisionMessage getLatestRevision(
         if (filter.equals("")) {
             filter = ".*";
         }
-        Pattern pattern = Pattern.compile(filter);
 
-        List<Object> matching = tags.stream().filter(pattern.asPredicate()).collect(Collectors.toList());
+        try {
+            Pattern pattern = Pattern.compile(filter);
 
-        if (matching.isEmpty()) {
-            LOGGER.info("Found no matching revision.");
-            return new PackageRevisionMessage();
-        }
+            List<Object> matching = tags.stream().filter(pattern.asPredicate()).collect(Collectors.toList());
 
-        String latest = "";
-        for (Object tag: matching) {
-            latest = biggest(latest, tag.toString());
-        }
+            if (matching.isEmpty()) {
+                LOGGER.info("Found no matching revision.");
+                return new PackageRevisionMessage();
+            }
 
-        LOGGER.info(String.format("Latest revision is: %s", latest));
-        return new PackageRevisionMessage(latest, new Date(), "docker", null,null);
+            String latest = "";
+            for (Object tag: matching) {
+                latest = biggest(latest, tag.toString());
+            }
+
+            LOGGER.info(String.format("Latest revision is: %s", latest));
+            return new PackageRevisionMessage(latest, new Date(), "docker", null,null);
+
+        } catch (PatternSyntaxException e) {
+            String message = String.format("Invalid docker tag filter '%s' used for image '%s': %s", filter, url, e.getMessage());
+            LOGGER.error(message);
```

Comment:
As you can see, making sure to do logging of the issue here, so it's also outputted in the plugin log when it happens. 😊